test/lit/binary/dylink.test

```diff
@@ -0,0 +1,10 @@
+# Verify that extra bytes in the dylink section are preserved.
+#
+# `dylink.test.wasm` was generated using emscripten to build side
+# module with two TLS exports (which adds extra information to the
+# dylink section) using:
+#  `emcc -s SIDE_MODULE side.c`
+# and then removing all sections except the dylink section using:
+#  `llvm-objcopy --only-section=dylink`
+RUN: wasm-opt -O1 %s.wasm -o %t.o
+RUN: cmp %s.wasm %t.o
```

Comment:
Is the source file short enough to quote here in the comment?

Reply:
At yes I can do that.